SKILL.tsv

```diff
@@ -3836,7 +3836,7 @@ SKILL_20150414_003835	{memo X}${#993399}{ol}[Magic] - [Holy]{/}{/}{nl}Sprinkle h
 SKILL_20150414_003836	{memo X}Create magic circle where ally's DEX increase and enemy's ecasion and Physical defense decrease.
 SKILL_20150414_003837	{memo X}Give blessings that grant additional damage on attacks of party members.
 SKILL_20150414_003838	$Resurrects a target that has been defeated in combat.
-SKILL_20150414_003839	$Temporarily grants the Holy property to you and your party member's attacks.
+SKILL_20150414_003839	$Temporarily grants an additional Holy property attack to you and your party member's attacks.
```

Comment:
The Korean description says that it temporarily grants the Holy property to both your attacks and that of your party members. What do you mean with 'additional'? From what I understand of the original, all the attacks you deal are rendered with the Holy property.